src/ripple/app/rdb/backend/RelationalDBInterfaceSqlite.cpp

```diff
@@ -257,23 +260,33 @@ class RelationalDBInterfaceSqliteImp : public RelationalDBInterfaceSqlite
     }
 
     /**
-     * @brief checkoutLedger Checks if node ledger DB exists.
+     * @brief existsLedger Checks if node ledger DB exists.
      * @return True if node ledger DB exists.
      */
     bool
     existsLedger()
     {
-        return !!lgrdb_;
+        return (bool)lgrdb_;
```

Comment:
the bool() operator automatically evaluates whether a unique_ptr contains something, so casting is redundant.

Also, the way to cast this if you really needed to would be by using something like static_cast<bool>(anotherpodtype);

Reply:
The compiler actually doesn't actually invoke the `bool` operator without the cast. Changed to `static_cast` instead.